bindings/java/src/lib.rs

```diff
@@ -88,6 +65,12 @@ pub unsafe extern "system" fn Java_org_apache_opendal_Operator_freeOperator(
     _class: JClass,
     ptr: *mut Operator,
 ) {
+    if ptr.is_null() {
+        // The pointer is null, nothing to free
```

Comment:
We should panic here to let user know: It's wrong to input NULL pointer into opendal's API.